### PR TITLE
Test public link share using email with email to self checked

### PR DESCRIPTION
**Pull request by skshetry**
## Description

This adds a test that the user and the creator of the link both gets the email if the `Send a copy to self` is checked.

## Related Issue

- Related to #30892 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [x] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [ ] Backport (if applicable set "backport-request" label and remove when the backport was done)

### CODE REVIEW

File: tests/acceptance/features/lib/FilesPageElement/SharingDialogElement/EditPublicLinkPopup.php [hunk 1]
```diff
@@ -41,6 +41,8 @@ class EditPublicLinkPopup extends OwncloudPage {
 	private $expirationDateLabelXpath = ".//label[contains(text(), 'Expiration')]";
 	private $expirationDateInputXpath = ".//input[contains(@class,'expirationDate')]";
 	private $emailInputXpath = "//form[@id='emailPrivateLink']//input[@class='select2-input']";
+	private $emailToSelfCheckboxXpath = "//form[@id='emailPrivateLink']" .
+										"//input[@class='emailPrivateLinkForm--emailToSelf']";
```
Comment (phil-davis) [inline tests/acceptance/features/lib/FilesPageElement/SharingDialogElement/EditPublicLinkPopup.php]: Why so much indent?

**Review by phil-davis — COMMENTED** [review]
Looks good. Just 1 indenting problem.

File: tests/acceptance/features/lib/FilesPageElement/SharingDialogElement/EditPublicLinkPopup.php (hunk 1 above)
Comment (skshetry) [reply tests/acceptance/features/lib/FilesPageElement/SharingDialogElement/EditPublicLinkPopup.php]: @phil-davis fixed.

**Review by phil-davis — APPROVED** [review]

**Comment by phil-davis** [comment]
@skshetry please backport

**Comment by phil-davis** [comment]
Backport ``stable10`` #32799

**Pull request merged**